BraintreeCore/src/main/java/com/braintreepayments/api/ConfigurationLoaderCallback.kt

```diff
@@ -1,5 +1,5 @@
 package com.braintreepayments.api
 
-internal interface ConfigurationLoaderCallback {
+internal fun interface ConfigurationLoaderCallback {
```

Comment:
:fire: nice catch on `fun` for interfaces

Reply:
Yeah I recently noticed it's a thing. We should be able to go 100% Kotlin in `v4` with functional interfaces.